lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -102,7 +102,7 @@ module Cardano.Wallet.Api.Link
     , getNetworkParams
     , getNetworkClock
     , getNetworkClock'
-
+    , getBlocksLatestHeader
```

Comment:
The name is awkward, it reads  as "Get block's latest header" whereas "Get latest block's header" is what we want corresponds to either `getLatestBlockHeader` or `getLatestBlock'sHeader`

Reply:
its `GET /blocks/latest/header` endpoint